server.go

```diff
@@ -230,6 +230,19 @@ func (s *Server) load() error {
 			}
 		}
 
+		// Open all shards.
+		for _, db := range s.databases {
+			for _, rp := range db.policies {
+				for _, g := range rp.shardGroups {
+					for _, sh := range g.Shards {
+						if err := sh.open(s.shardPath(sh.ID)); err != nil {
```

Comment:
we should only try to open the shard if the metadata for it says it exists on this server id